<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;MS Reference Sans Serif&quot;;
background: rgb(105, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #007cFF;
background: #333333;
color: #FFF;</string>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid #007cFF;
background: #333333;
color: #FFF;</string>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>125</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="maximumSize">
      <size>
       <width>84</width>
       <height>125</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Vladimir Script&quot;;
background:rgba(0, 0, 0, 120)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>Фото</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">text-decoration: underline;
color: white</string>
          </property>
          <property name="text">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string>Любимое блюдо</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">color: white</string>
         </property>
         <property name="text">
          <string>м</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="styleSheet">
          <string notr="true">color: white</string>
         </property>
         <property name="text">
          <string>ж</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
